components/calendar/appearance.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Appearance
+page_title: Calendar Appearance
+description: Appearance settings of the Calendar for Blazor.
+slug: calendar-appearance
+tags: telerik,blazor,calendar,appearance
+published: True
+position: 30
+---
+
+# Appearance Settings
+
+You can control the appearance of the Calendar by setting the following attribute:
+
+## Size
```

Comment:
It's good to explain what does this parameter change and what it doesn't. Some customers have incorrect expectations.